gds_py/Dockerfile

```diff
@@ -40,7 +43,12 @@ RUN conda install -c conda-forge -c defaults --quiet --yes \
      'statsmodels' \
      'urbanaccess' \
      'xlrd' \
-     'xlsxwriter'
+     'xlsxwriter' \
+     && conda clean --all --yes --force-pkgs-dirs \
+     && find /opt/conda/ -follow -type f -name '*.a' -delete \
+     && find /opt/conda/ -follow -type f -name '*.pyc' -delete \
+     && find /opt/conda/ -follow -type f -name '*.js.map' -delete \
+     && conda list
 
 # pip commands
 RUN pip install -U --no-deps pysal==2.1.0
```

Comment:
All but `bambi` are in `conda-forge`, any reason to `pip` install them?

PS: https://github.com/conda-forge/staged-recipes/pull/9053